### PR TITLE
Update mobile UI screenshots and consolidate docs

### PR DESCRIPTION
**Pull request by ghost**
I consolidated content that was spread out over a few pages and updated screenshots to reflect newest UI changes.

### CODE REVIEW

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx [hunk 1]
```diff
@@ -1,12 +1,14 @@
 ---
-title: 'Handled exceptions: Analyze trends, prevent crashes'
+title: 'Handled exceptions: Analyze trends, prevent crashes, view specific occurances'
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx:2]:
```suggestion
title: 'Handled exceptions: Analyze trends, prevent crashes, drill down into occurrences'
```

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx [hunk 2]
```diff
@@ -17,21 +19,13 @@ import mobileHandledExceptionGroups from 'images/mobile_screenshot-full_handled-
...
-2. Use any of New Relic's [standard page functions](/docs/data-analysis/user-interface-functions/view-your-data/standard-page-functions) to drill down into detailed information; for example, zoom into any area of a chart.
-3. Look for obvious or general trends in the **Users affected** and **Sessions affected** [percentage charts](#percentage-charts).
-4. Adjust the types of exceptions shown by using [groups and filters](#groups-filters).
-5. Optional: [Query or share the chart data](#insights).
-6. Look for similar patterns where exceptions appear in stack traces with the [**Top 5 exception locations** table](#top5-exceptions).
-7. To view stack trace [thread details for each occurrence](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-occurrences) of the exception, select a record from the **Top 5 exceptions location** table.
+Our **Handled exceptions** feature help you identify significant factors contributing to poor mobile application experience, and use filterable data to find a resolution more quickly. You can also use the [handled exceptions API](#hex-api) to customize the data you send, and use NRQL to query and share the data.
```
Comment (paperclypse) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx:28]: Should it be "helps you"?

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx [hunk 3]
```diff
@@ -43,9 +37,31 @@ To get the most out of the **Handled exceptions** UI, use this basic workflow:
   **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Exceptions > Handled exceptions:** As you explore the wealth of data in the charts and table, use groups and filters to discover patterns that help you determine the root cause of mobile app exceptions.
```
Comment (paperclypse) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx:37]: Nice caption text!

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx (hunk 2 above)
Comment (paperclypse) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx:28]: I think "handled exceptions" should be lowercase. 
https://docs.newrelic.com/docs/style-guide/capitalization/product-capability-feature-usage/

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx [hunk 4]
```diff
@@ -43,9 +37,31 @@ To get the most out of the **Handled exceptions** UI, use this basic workflow:
...
 </figcaption>
 
+## Features
+
+Handling exceptions as they occur can help improve your mobile app users' experience, but it's not enough to catch exceptions. You also need to know how to prevent them. Our Handled exceptions feature helps you answer these kinds of questions:
+
+* How many different types of handled exceptions are occurring? A high occurrence rate may necessitate changes to the back-end systems.
```
Comment (paperclypse) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx:44]: It's "backend" https://docs.newrelic.com/docs/style-guide/word-choice/usage-dictionary/#backend

Also, maybe make that sentence more active? Something like "When you're seeing a lot of handled exceptions, you may need to make changes to your backend systems"

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx [hunk 5]
```diff
@@ -43,9 +37,31 @@ To get the most out of the **Handled exceptions** UI, use this basic workflow:
...
+By using handled exceptions, you can identify and resolve these kinds of issues more quickly.
+
+## Requirements
+
+To view handled exception data, your app needs to meet these requirements:
+
+* **Android:** [Android agent version 5.15.0 or higher](/docs/release-notes/mobile-release-notes/android-release-notes)
+* **iOS:** [iOS agent version 5.15.0 or higher](/docs/release-notes/mobile-release-notes/ios-release-notes)
+
+Note that how long your mobile data is available for depends on the data retention you've set for mobile data.
```
Comment (paperclypse) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx:58]: I'm not sure how much "note that" adds to a sentence :)

**Review by paperclypse — DISMISSED** [review]
Had some minor comments, but on the whole I think this is a solid improvement. Nice work consolidating these docs!

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx (hunk 2 above)
Comment (ally-sassman) [reply src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx:28]: @paperclypse But it's the name of the UI... still think it should be lowercased?
image

Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx:28]:
```suggestion
Our **Handled exceptions** feature helps you identify significant factors contributing to poor mobile application experience, and use filterable data to find a resolution more quickly. You can also use the [handled exceptions API](#hex-api) to customize the data you send, and use NRQL to query and share the data.
```

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx (hunk 4 above)
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx:44]:
```suggestion
* How many different types of handled exceptions are occurring? A high occurrence rate may indicate you need to make changes to the back-end systems.
```

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx [hunk 6]
```diff
@@ -43,9 +37,31 @@ To get the most out of the **Handled exceptions** UI, use this basic workflow:
...
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx:58]:
```suggestion
```

File: src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx (hunk 2 above)
Comment (paperclypse) [reply src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx]: Yeah, it should still be lowercased unless you are specifically referring to the UI in a procedure, such as "click *Handled exceptions*, then such and such"

When just referring to it more broadly, it should be lowercase

Comment (ally-sassman) [reply src/content/docs/mobile-monitoring/mobile-monitoring-ui/crashes/handled-exceptions-analyze-trends-prevent-crashes.mdx]: Ah I see. Okay I'll fix it in another PR.

**Pull request merged**